google/cloud/security/scanner/scanner.py

```diff
@@ -333,12 +336,12 @@ def _build_scan_summary(all_violations, total_resources):
 
         # Keep track of # of violations per resource id.
         if (violation.resource_id not in
-            resource_summaries[resource_type]['violations']):
+                resource_summaries[resource_type]['violations']):
             resource_summaries[resource_type][
                 'violations'][violation.resource_id] = 0
 
         # Make sure to count each member violation as a separate one.
-        for member in violation.members:
+        for _ in violation.members:
```

Comment:
Unrelated to this change, but this should be refactored to remove the loop.

         members = len(violation.members)    
         resource_summaries[resource_type][
              'violations'][violation.resource_id] += members
  
         total_violations += members